backend/pvm.pm

```diff
@@ -45,11 +45,6 @@ sub do_start_vm {
     return {};
 }
 
-sub runcmd {
```

Comment:
+1 for grouping common function, but afaics from the changes it's used in backends, probably best place for it should be in [baseclass](https://github.com/os-autoinst/os-autoinst/blob/master/backend/baseclass.pm)

Reply:
Note aside: as functions in osutils are meant to be re-used, they should come up with unit tests

Reply:
I moved it to baseclass